docs/develop/create-and-debug-office-add-ins-in-visual-studio.md

```diff
@@ -99,6 +100,12 @@ You can use Visual Studio to debug your add-in in the Office desktop client on W
 > [!NOTE]
 > You cannot use Visual Studio to debug add-ins in Office on the web or Mac. For information about debugging on these platforms, see [Debug Office Add-ins in Office on the web](../testing/debug-add-ins-in-office-online.md) or [Debug Office Add-ins on iPad and Mac](../testing/debug-office-add-ins-on-ipad-and-mac.md)
 
+### Enable debugging for add-in commands and UI-less code
+
+When Visual Studio debugs Office on Windows, the add-in is hosted in either a Microsoft Internet Explorer or Microsoft Edge browser instance. To determine which browser is being used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

Comment:
Since this is merged, I can't "suggest." But you can remove "Microsoft" and just say "Internet Explorer (IE) or Edge browser instance."